.circleci/test.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# This script runs all tests in glow, including onnxifi gtests
+
+
+set -euxo pipefail
+
+GLOW_DIR=$PWD
+GLOW_BUILD_DIR=${GLOW_DIR}/build
+cd ${GLOW_BUILD_DIR}
```

Comment:
noob question, how do we ensure that this folder exists? (part of the build script i guess)

Reply:
yeah, check this in build.sh:
# Build glow
  cd ${GLOW_DIR}
  mkdir build && cd build